libs/vkd3d/vkd3d_private.h

```diff
@@ -406,6 +406,102 @@ struct d3d12_fence
 HRESULT d3d12_fence_create(struct d3d12_device *device,
         uint64_t initial_value, D3D12_FENCE_FLAGS flags, struct d3d12_fence **fence);
 
+enum vkd3d_allocation_flag
+{
+    VKD3D_ALLOCATION_FLAG_GLOBAL_BUFFER     = (1u << 0),
+    VKD3D_ALLOCATION_FLAG_DEDICATED_BUFFER  = (1u << 1),
```

Comment:
I'm a little confused about what the difference between DEDICATED and GLOBAL is here.

Reply:
GLOBAL means that the allocation owns a buffer, whereas DEDICATED only references an existing buffer.